frontend/components/Footer.tsx

```diff
@@ -94,26 +94,17 @@ export default function SmallWithLogoLeft() {
           <Link href={'/feedback'}>Feedback</Link>
           <Box minWidth={'170'}>
             <Select
+              defaultValue={getNetworkConfig(process.env.NEXT_PUBLIC_DEFAULT_NETWORK!).chainId}
```

Comment:
Fixes a console log where it suggested to use "defaultValue" instead of "selected"